package.json

```diff
@@ -5,17 +5,17 @@
   "author": "Dustin Schau <dustinschau@gmail.com> (https://dustinschau.com)",
   "repository": "https://github.com/dschau/gatsby-blog-starter-kit",
   "dependencies": {
-    "gatsby": "~1.9.45",
-    "gatsby-link": "~1.6.17",
-    "gatsby-plugin-catch-links": "~1.0.1",
-    "gatsby-plugin-react-helmet": "~1.0.1",
-    "gatsby-plugin-react-next": "~1.0.4",
-    "gatsby-plugin-sharp": "~1.6.48",
-    "gatsby-remark-images": "~1.5.13",
-    "gatsby-source-filesystem": "~1.5.1",
-    "gatsby-transformer-remark": "~1.7.12",
-    "react": "~16.2.0",
-    "react-dom": "~16.2.0",
+    "gatsby": "2.0.0-beta.34",
+    "gatsby-link": "2.0.0-beta.4",
+    "gatsby-plugin-catch-links": "2.0.2-beta.2",
+    "gatsby-plugin-react-helmet": "3.0.0-beta.3",
+    "gatsby-plugin-sharp": "2.0.0-beta.5",
+    "gatsby-remark-images": "2.0.1-beta.6",
+    "gatsby-source-filesystem": "2.0.1-beta.3",
+    "gatsby-transformer-remark": "2.1.1-beta.3",
+    "react": "~16.3.0",
+    "react-dom": "~16.3.0",
+    "react-helmet": "~5.2.0",
```

Comment:
Note: some plugins (e.g. `gatsby-plugin-react-helmet`) no longer depend on peer dependencies, e.g. `react-helmet`. As such, I had to manually install the `react-helmet` dependency.